.gitignore

```diff
@@ -1,7 +1,10 @@
+data
```

Comment:
There are files under data that are in github. (For example [data/spell/words.txt](https://github.com/mozilla/DeepSpeech/tree/master/data/spell)) Why should we ignore the data directory?